### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by SapkaleBharti**
Please review my changes file

### CODE REVIEW

File: DHL/Dynamic json/xc_en.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                "formatHint": "Upload file(s)",
+                "description": "Please upload a copy of your original invoice for the customs clearance of this shipment.",
+                "toolTip": ""
+            },
+            "PerMedCheckBx": {
+                "description": "I understand that beside the requested documents, I also have to deliver the original doctor's prescription to the DHL offices located at Schottegatweg Oost 18.",
+                "title": "Personal Medicines",
+                "toolTip": "To be delivered at our counter at Schottegatweg Oost 18"
+            },
+            "description": "",
```
Comment (mevijaykakade90) [inline DHL/Dynamic json/xc_en.json:15]: add parent attribute in single line.

File: DHL/Dynamic json/xc_es.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                "formatHint": "Cargar archivo(s)",
+                "description": "Cargue una copia de su Factura Original para el despacho de aduana de este envío.",
+                "toolTip": ""
+            },
+            "PerMedCheckBx": {
+                "description": "Entiendo que además de los documentos solicitados, también debo entregar la receta médica original en las oficinas de DHL ubicadas en Schottegatweg Oost 18.",
+                "title": "Medicamentos Personales",
+                "toolTip": "Para ser entregado en nuestro mostrador en Schottegatweg Oost 18"
+            },
+            "description": "",
```
Comment (mevijaykakade90) [inline DHL/Dynamic json/xc_es.json:15]: add parent attributes in single line.

File: DHL/Dynamic json/xc_en.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            "toolTip": "",
+            "StampInvImg": {
+                "formatHint": "Upload file(s)",
+                "description": "Please upload a copy of your Stamp Invoice for the customs clearance of this shipment.",
+                "toolTip": "",
+                "title": "Stamp Original Invoice"
+            },
+            "ShipDHLRadioBtn": {
+                "description": "Would you like to clear your shipment with DHL?<br>1. Yes<br>2. No",
```
Comment (mevijaykakade90) [inline DHL/Dynamic json/xc_en.json:24]: Radio button should be proper.
Example : "description": "Would you like to clear your shipment with DHL?",
                "Opt1": "Yes",
                "Opt2": "No",

File: DHL/Dynamic json/xc_es.json (hunk 2 above)
Comment (SapkaleBharti) [reply DHL/Dynamic json/xc_es.json:15]: Done

File: DHL/Dynamic json/xc_en.json (hunk 3 above)
Comment (SapkaleBharti) [reply DHL/Dynamic json/xc_en.json:24]: Done

**Pull request merged**